extensions/LIZARD_OFFICIAL/aes.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable */
```

Comment:
Same thing here about `/* eslint-disable */`, see previous comment pertaining to it.